### PR TITLE
Contact Us

### PR DESCRIPTION
**Pull request by YuriGizra**
#97

### CODE REVIEW

**Comment by niryariv** [comment]
@YuriGizra please add a screenshot of the page

File: ibli/modules/custom/ibli_general/ibli_general.module [hunk 1]
```diff
@@ -14,3 +14,69 @@ function ibli_general_ctools_plugin_directory($module, $plugin) {
     return 'plugins/' . $plugin;
   }
 }
+
+/**
+ * Form builder; Contact form.
+ */
+function ibli_general_contact_form() {
+  $form = array();
```
Comment (amitaibu) [inline ibli/modules/custom/ibli_general/ibli_general.module]: @NofarGizra 
We shouldn't do a custom form -- we can use the regular contact module.

**Comment by YuriGizra** [comment]
Update:
- Added address and map on sidebar.
- Added contact form.

Still ToDo:
- Form functionality.
- Page element with form description.
- Design.

**Comment by YuriGizra** [comment]
selection_003

**Comment by niryariv** [comment]
this seems to be missing the top part from the bootstrap theme - note the original design:

screen shot 2014-06-08 at 11 06 58 am

**Comment by NofarGizra** [comment]
@niryariv 
This is still "work in progress" (WIP). The design will come later.

**Comment by NofarGizra** [comment]
Update:
- Contact form works.
- Top menu: Added link to contact form.
- Design!

**Comment by NofarGizra** [comment]
**Orange arrow:** Added link to contact us page.
**Purple arrow:** Editable content (form description).
**Purple arrow:** Editable content (iframe of map).
**Green arrows:** Editable content (contact address), note that it is **also displayed in the footer**.

contactus

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**